lifecycle/lifecycle/deployer/provision.py

```diff
@@ -110,10 +114,20 @@ def on_job_alive():
     update_job(job)
 
     metric_deployed_job.inc()
-    logger.info(f'job {manifest.name} v{manifest.version} has been provisioned, deployment ID: {deployment.id}')
+    logger.info(f'job {manifest.name} {manifest.version} has been provisioned, deployment ID: {deployment.id}')
```

Comment:
Assuming this change is because `manifest.version` already has a v in it? (Or something similar that makes it obvious it's a version number?)

If not, I'd prefer the v to stay. It makes it a little clearer in my opinion.

This comment applies to the below lines logging `{job.name} {job.version}` too.

Reply:
A while ago I noticed that `{name} v{version}` doesn't work well with version aliases like `latest` which is then transformed into `adder vlatest`, that looks stupid. It's been in many places, that's why I amend this whenever I see it.
But actually, there's no risk of it in this particular line as it makes sense for Job class, not for the Manifest class which always should have exact version. 
However, Manifest class has another version field that is vulnerable - `jobtype_version`, it may contain `latest`.
That's why I'm generally striving to unify this and I think `{name} {version}` format is the best idea anyway.